files/ru/web/svg/attribute/fill-rule/index.md

```diff
@@ -40,22 +47,34 @@ svg {
 </svg>
 ```
 
-{{EmbedLiveSample('topExample', '100%', 200)}}
+{{EmbedLiveSample("Example", '100%', 200)}}
```

Comment:
Please use the translated headings instead

```suggestion
{{EmbedLiveSample("Пример", '100%', 200)}}
```

Reply:
@yin1999 thank you! 
You are absolutely right!
I missed this place.